types/recharts/index.d.ts

```diff
@@ -426,7 +427,7 @@ export interface RadarProps {
 	label?: boolean | any | Element | RechartsFunction;
 	isAnimationActive?: boolean;
 	animationBegin?: number;
-	animationDuration?: number;
+	// animationDuration?: number;
```

Comment:
If the property is not there any more, could you please *remove* it instead of commenting it out?